src/index.html

```diff
@@ -21,7 +21,9 @@
     passageEndpoint: 'http://localhost:5001',
     environment: 'development',
     version: 'development',
-    ingressBaseDomain: 'k8s.sample.io'
+    ingressBaseDomain: 'k8s.sample.io',
+    awsCapabilitiesJSON: '',
+    azureCapabilitiesJSON: '',
   };
   window.config = config;
   </script>
```

Comment:
Out of curiosity, why do you define the config in the html?

Reply:
I don't have a better way to get config into happa.

Happa is just static html files behind nginx, nothing dynamic going on.

We have a `start.sh` which then replaces these values with things it finds in the ENV.

If happa had some serverside component, then I could pass it in as parameters to that binary, but that is not the case.